laske_export/document/invoice_sales_order_adapter.py

```diff
@@ -264,24 +278,25 @@ def get_line_items(self):
 
         return line_items
 
-    def get_order_type(self):
+    def get_order_type(self) -> str | None:
         if self.invoice.type == InvoiceType.CHARGE:
             return "ZTY1"
         elif self.invoice.type == InvoiceType.CREDIT_NOTE:
             return "ZHY1"
 
-    def get_original_order(self):
+    def get_original_order(self) -> str | None:
         if self.invoice.type == InvoiceType.CREDIT_NOTE:
             return str(self.invoice.credited_invoice.number)
 
-    def get_sales_office(self):
+    def get_sales_office(self) -> str:
         if self.invoice.lease.lessor and self.invoice.lease.lessor.sap_sales_office:
             return self.invoice.lease.lessor.sap_sales_office
 
-        # TODO: Remove
-        return "2826"
+        # TODO: What should be the default value when SAP sales office is not found?
+        #       Make, or something else? Maybe return empty string instead?
```

Comment:
Raising some kind of error probably?

Reply:
Maybe yes. I'm unsure whether it is better to pass an empty string to the invoice so that customer knows something is wrong, or raise an error and block the export from being sent.